src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -3707,4 +3707,10 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="PreferencesViewIncludeTimestampExportPathTooltip" xml:space="preserve">
     <value>When toggled on, file names of exported images include date and time of export.</value>
   </data>
+  <data name="ElementBindingWarningMessage" xml:space="preserve">
+    <value>A Save As command will treat the workspace as a completely new file and cause element binding to break. Choose Save to preserve element binding with the host document.</value>
+  </data>
+  <data name="ElementBindingWarningTitle" xml:space="preserve">
+    <value>Element Binding Warning</value>
+  </data>
```

Comment:
Not sure why adding these resources is causing this exception to be thrown: 
![image](https://github.com/DynamoDS/Dynamo/assets/5710686/1f0c3156-cd71-41c8-9bd5-ca3d52634810)